protocol/v2/testing/test_utils.go

```diff
@@ -152,3 +160,63 @@ func CommitDataToBytes(t *testing.T, input *specqbft.CommitData) []byte {
 	require.NoError(t, err)
 	return ret
 }
+
+func GetSpecTestJSON(path string, module string) ([]byte, error) {
+	fileName := "tests.json"
+	filePath := path + "/" + fileName
+	rootPath := path
+	for {
+		if _, err := os.Stat(filepath.Join(rootPath, "go.mod")); err == nil {
+			break
+		}
+		rootPath = filepath.Dir(rootPath)
+	}
+	buf, err := os.ReadFile(fmt.Sprintf("%s/go.mod", rootPath))
+	if err != nil {
+		return nil, errors.New("could not read go.mod")
+	}
+	goModFile, err := modfile.Parse("go.mod", buf, nil)
+	if err != nil {
+		return nil, errors.New("could not parse go.mod")
+	}
+	var req *modfile.Require
+	for _, r := range goModFile.Require {
+		if strings.EqualFold("github.com/bloxapp/ssv-spec", r.Mod.Path) {
+			req = r
+			break
+		}
+	}
+	if req == nil {
+		return nil, errors.New("could not find ssv-spec module")
+	}
+	var version string
+	splitModVersion := strings.Split(req.Mod.Version, "-")
+	if len(splitModVersion) > 1 {
+		version = splitModVersion[len(splitModVersion)-1]
+	} else {
+		version = splitModVersion[0]
+	}
+
+	resp, err := http.Get(fmt.Sprintf("https://raw.githubusercontent.com/bloxapp/ssv-spec/%s/%s/spectest/generate/tests.json", version, module))
```

Comment:
Downloading the json during test might make it unstable, why not downloading it in a dedicated make cmd?